package.json

```diff
@@ -61,9 +61,11 @@
     "sanitize-filename": "^1.6.1",
     "sinon": "^1.17.6",
     "socket.io": "^1.3.2",
+    "stack-trace": "0.0.10",
     "supertest": "^3.0.0",
     "underscore": "^1.8.3",
     "url": "^0.11.0",
+    "util": "^0.10.3",
```

Comment:
Is the util module not included in Node by default and therefore doesn't need explicit installation as a package?